### ISSUE TITLE
Push view controller from app delegate with navigation controller

### ISSUE THREAD

**Issue by quetool** [John-Lluch/SWRevealViewController#588]
Hi, i want to push a view controller with rootNavigationController from app delegate but i get an error

[SWRevealViewController pushViewController:animated:]: unrecognized selector sent to instance 0x13f08b200

This is what i do:

UINavigationController *navigationController = (UINavigationController *)self.window.rootViewController;
MyViewController *controller = (MyViewController *)[storyboard instantiateViewControllerWithIdentifier:@"MyViewController"];
[navigationController pushViewController:controller animated:YES];

**Comment by iDevelopper** [comment]
self.window.rootViewController is a SWRevealViewController!

**Comment by quetool** [comment]
yes, thanks! we continue from here https://github.com/John-Lluch/SWRevealViewController/issues/584

**Comment by shruezee** [comment]
if anyone have issues with ios 13 using present vc or push vc, following way solves. 
CustomViewController* c_vc = [[CustpmViewController alloc] init];
            if (@available(iOS 13.0, *)) {
                login_vc.modalPresentationStyle = UIModalPresentationFullScreen;
            }
                [self presentViewController:c_vc animated:YES completion:nil];
